satellite/overlay/service.go

```diff
@@ -114,6 +119,17 @@ type UpdateRequest struct {
 	UptimeDQ     float64
 }
 
+// ExitStatusRequest is used to update a node's graceful exit status.
+type ExitStatusRequest struct {
+	NodeID              storj.NodeID
+	ExitInitiatedAt     *time.Time
+	ExitLoopCompletedAt *time.Time
+	ExitFinishedAt      *time.Time
+	UpdateInitiated     bool
+	UpdateLoopCompleted bool
+	UpdateFinished      bool
```

Comment:
I'm wondering if we need these?  Could you just check IsZero on the time and if it's not zero, update the field. Otherwise leave as is since we shouldn't be un-setting values.

Reply:
okay I think that makes sense!

Reply:
I ended up checking if not nil instead since it's a pointer, hopefully that will still work 👍